src/autogluon/bench/cloud/aws/destroy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+STATIC_RESOURCE_STACK_NAME=$1
+BATCH_STACK_NAME=$2
+
+
+# Get the stack outputs JSON object
+OUTPUTS_JSON=$(aws cloudformation describe-stacks --stack-name "${BATCH_STACK_NAME}" --query "Stacks[0].Outputs" --output json)
+
+REPOSITORY_NAME=$(echo "${OUTPUTS_JSON}" | jq -r '.[] | select(.OutputKey == "EcrRepositoryName") | .OutputValue')
+IMAGE_URI=$(echo "${OUTPUTS_JSON}" | jq -r '.[] | select(.OutputKey == "ImageUri") | .OutputValue')
```

Comment:
I think this does the same thing is a more simple way:
`jq -r '.[].OutputValue | select(.OutputKey == "EcrRepositoryName")'
`

Reply:
The original command might be slightly more efficient when more keys are present since it filters the output to only the required key-value pair before returning the value.